docs/developer-guide/ksqldb-reference/select-pull-query.md

```diff
@@ -14,98 +14,130 @@ Synopsis
 the API responds with a single response.  
 
-The WHERE clause must contain a value for each primary-key column to retrieve and may
-optionally include bounds on `WINDOWSTART` and `WINDOWEND` if the materialized table is windowed.
-For more information, see 
-[Time and Windows in ksqlDB](../../concepts/time-and-windows-in-ksqldb-queries.md).
-
-Example
--------
-
-```sql
-SELECT * FROM pageviews_by_region
-  WHERE regionId = 'Region_1'
-    AND 1570051876000 <= WINDOWSTART AND WINDOWEND <= 1570138276000;
-```
-
-If the `pageviews_by_region` table was created as an aggregation of multiple columns,
-then each key column must be present in the WHERE clause. The following example shows how to 
-query the table if `countryId` and `regionId` where both key columns:
 
-```sql
-SELECT * FROM pageviews_by_region
-  WHERE countryId = 'USA' AND regionId = 'Region_1'
-    AND 1570051876000 <= WINDOWSTART AND WINDOWEND <= 1570138276000;
-```
-
-When writing logical expressions using `WINDOWSTART` or `WINDOWEND`, you can use ISO-8601
-formatted datestrings to represent date times. For example, the previous
-query is equivalent to the following:
-
-```sql
-SELECT * FROM pageviews_by_region
-  WHERE regionId = 'Region_1'
-    AND '2019-10-02T21:31:16' <= WINDOWSTART AND WINDOWEND <= '2019-10-03T21:31:16';
-```
-
-You can specify time zones within the datestring. For example,
-`2017-11-17T04:53:45-0330` is in the Newfoundland time zone. If no time zone is
-specified within the datestring, then timestamps are interpreted in the UTC
-time zone.
-
-If no bounds are placed on `WINDOWSTART` or `WINDOWEND`, rows are returned for all windows
-in the windowed table.
-
-Also, you can issue a pull query against a derived table that was created by using the [CREATE TABLE AS SELECT](../../ksqldb-reference/create-table-as-select) statement. 
+-   Pull queries are expressed using a strict subset of ANSI SQL.
+-   You can issue a pull query against any table that was created by using 
+    a [CREATE TABLE AS SELECT](../../ksqldb-reference/create-table-as-select) 
+    statement.
+-   `SELECT` statements can contain column arithmetic and function calls.
+-   Currently, we do not support pull queries against tables created by using a [CREATE TABLE](../../ksqldb-reference/create-table) statement.
+-   `JOIN`, `PARTITION BY`, `GROUP BY` and `WINDOW` clauses aren't supported currently.
+
+WHERE Clause Guidelines
+-----------------------
+
+By default, only key lookups are enabled. They have the following requirements:
+-   Key column(s) must use an equality comparison to a literal (e.g. KEY = 'abc').
+-   On windowed tables, WINDOWSTART and WINDOWEND can be optionally compared to literals. 
+    For more information on windowed tables, see [Time and Windows in ksqlDB](../../concepts/time-and-windows-in-ksqldb-queries.md).
+
+You can enable table scans to loosen the restrictions on the `WHERE` clause or eliminate 

```

Comment:
we might want to say what a "table scan" is (perhaps the note below is a good place for this). @JimGalasyn thoughts?